### ISSUE TITLE
Why VL have been disabled for Swipeable tabs?

### ISSUE THREAD

**Issue by Pitouli** [framework7io/framework7#3649]
Hello,

Since v5.7.5, my Virtual List stoped to work: it shows only the first element.

Story time: _I thought it would be a good idea to analyze the issue myself and make a PR, and after a deep search, I finally found the issue: the size of the hidden VL was null when being generated, so the algorithm considered that the first element was enough to show. And since clicking this unique element to change page and go back was making the VL work again, I understood the issue. the page:show event was refreshing the page, and I simply had to add an event when changing tabs to do the same. So I crawled the F7 code to find where to add this event. And I found. But when I was about to edit, I saw something strange..._ 

I thought it was a regression, but in fact you made a commit specially to disable VL in animated tabs: https://github.com/framework7io/framework7/commit/3ccef5e4d505c9738a68b9b17fd35d256e03b139

Since I suppose there is reasons for such an evolution, I suppose the best to do is remove the "animated" feature, even if it's sad to lose this...
But can you explain the reason of this evolution? Performance issues? Edge case? 

Thank you! :)

**Comment by nolimits4web** [comment]
What is your full layout? It is not needed on animated tabs show because tabs are not get hidden so styles recalculation is not required

**Comment by Pitouli** [comment]
I have animated tabs inside a view which is itself a tab of a tabbed-layout.

I reproduced it here: 
https://jsfiddle.net/9yo1btgw/3/

Go to Tab 3 of Tab B. 
The VL does not show any elements even if it knows that there is 10,000 elements (it sizes correctly the list), because it believes the page height is 0.
Remove the animated property and it works. 

_(Note : in my case, I see the first element. But it's probably a side effect of using Large Title and subnavbar...)_
